NEWcafelatte/cafelatte.py

```diff
@@ -13,4 +14,3 @@ def prepare_coffee(self):
     def prepare_coffee_time(self):
         time_to_prepare = 4 * self.coffee_spoons + 2 * self.sugar_spoons + self.milk/100
         print(f'dureaza {int(time_to_prepare)} secunde sa va preparam cafeaua, b0$$')
-
```

Comment:
b0$$, why didn't you use complex calculations (math functions)?

Reply:
b0$$, pt mine si adunarea e complexa xD